annif/transformer/inputlimiter.py

```diff
@@ -0,0 +1,22 @@
+# TODO Add docstring
+from annif.exception import ConfigurationException
+from . import transformer
+
+
+class InputLimiter(transformer.IdentityTransformer):
+
+    name = 'limit_input'
```

Comment:
The name is now hardcoded in two places: here an in `__init__.py`. It would be better to set it just once here and construct the dict in `init.py` using this information.